spec/fixtures/custom_nginx.template

```diff
@@ -312,60 +312,157 @@ http {
 
     server {
         listen 55555;
```

Comment:
In the future, we should probably change this port to something less cumberstone, like `9000+`... (`+` for replicas of this mock upstream for the LB tests)